client/src/pages/AdvancedSearch/AdvancedSearch.tsx

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import { useForm } from "./useForm";
+import {
+  AdvancedSearchWrapper,
+  Keyword,
+  Location,
+  Price,
+  Open,
+  Distance,
+  Form,
+} from "./AdvancedSearchStyles";
+import price_icon from "../../assets/images/price_icon.png";
+
+const AdvancedSearch = () => {
+  // getting the event handlers from our custom hook
+  const { onChange, onSubmit, values } = useForm(advancedSearchCallback);
+  // a submit function that will execute upon form submission
+  async function advancedSearchCallback() {
+    console.log(values);
+  }
+
+  return (
+    <AdvancedSearchWrapper>
+      <Form onSubmit={onSubmit}>
+        <fieldset>
+          <legend>
+            <h1>AdvancedSearch</h1>
+          </legend>
+          <Keyword>
+            <h2>Keyword</h2>
+            <input
+              name="keyword"
+              className="keyword-input"
+              onChange={onChange}
+              placeholder="vegan, gluten-free, etc."
+            />
+          </Keyword>
+
+          <Location>
+            <h2>Location</h2>
+            <input
+              name="location"
+              className="location-input"
+              onChange={onChange}
+              placeholder="city, state, zip"
+            />
+            <input
+              type="radio"
+              onChange={onChange}
+              id="current-radio"
+              value="current location data"
+            />
+            <label htmlFor="current-radio">Use Current</label>
```

Comment:
This should be a button, rather than a radio input. It will run the geo location function that I can add later.